### ISSUE TITLE
Rework Mountains algortithm

### ISSUE THREAD

**Issue by Ad5001** [Ad5001/BetterGen#19]
We need to rework the Mountain algorithm to make it look like real montains.

**Comment by AvgZing** [comment]
🤷‍♂️

**Comment by DerddyDert** [comment]
Im glad to see this. Thats one thing I really miss is decent mountains to go with all this.

**Comment by Vaults1** [comment]
🤷‍♂️

**Comment by AvgZing** [comment]
Man shrugging. Github preset emoji

**Comment by AvgZing** [comment]
Man shrugging. Github preset emojis don't work very well on mobile

**Comment by HimbeersaftLP** [comment]
> Man shrugging. Github preset emojis don't work very well on mobile

I don't see any issue here...

![Screenshot](https://cloud.githubusercontent.com/assets/16354747/26062949/c2b9d2de-398c-11e7-8dc5-4adf88b2af22.png)

**Comment by AvgZing** [comment]
android? mobile? For me on my galaxy SIII it looks like a box with an X but on computer its fine

**Comment by HimbeersaftLP** [comment]
What Android Version do you have?
They show up fine on both my Win10 Mobile Phone and my Android 7 tablet...

**Comment by inxomnyaa** [comment]
does this really matter soooo much to spam?

**Comment by AvgZing** [comment]
@HimbeersaftLP Cyanogen Mod 12.1 (CM12.1) which is equivalent to android 5.1
@thebigsmileXD Kind of, yes...

**Comment by jasonw4331** [comment]
Make a chat on gitter if you're going to spam @TheRoyalBlock @HimbeersaftLP

**Comment by HimbeersaftLP** [comment]
My last message about this:

> @HimbeersaftLP Cyanogen Mod 12.1 (CM12.1) which is equivalent to android 5.1

Unicode 9, the Unicode version that this Emoji was added in, has been added to Android in Version 7.0

**Comment by AvgZing** [comment]
Right. So my system is too outdated. That was my guess, but I was not entirely sure.
P.S to everyone else: Don't worry, this conversation is over. No need to worry about the spam any longer.
